extensions/remove-decorators/src/removeDecoratorsClipboard.tsx

```diff
@@ -0,0 +1,6 @@
+import { Clipboard } from "@raycast/api";
+import removeDecorators from "./utils/removeDecorators";
+
+export default async () => {
+  await removeDecorators((await Clipboard.readText()) || "");
```

Comment:
you could also use [getSelectedText](https://developers.raycast.com/api-reference/environment#getselectedtext) – should improve UX 😃